### DISCUSSION TITLE
[REQ]  Compatibility with RGT cycling

### DISCUSSION THREAD

**Discussion by HilmiiKumdakci** [cagnulein/qdomyos-zwift#288] [Q&A]
Hi,

RGT cycling is a zwift alternative and has a feature of "magic roads", which enables you to create your own road by uploading gpx file into their system and make your practice more on-purpose. I wonder if anyone tried qdomyos-zwift with RGT ? Does it work ?

**Comment by cagnulein** [comment]
Yes a lot of qz users use rgt. Search qdomyoszwift rgt on YouTube:)

On Sat, 24 Apr 2021 at 14:01, HilmiiKumdakci ***@***.***>
wrote:

> Assigned #287 <https://github.com/cagnulein/qdomyos-zwift/issues/287> to
> @cagnulein <https://github.com/cagnulein>.
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/cagnulein/qdomyos-zwift/issues/287#event-4639879673>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAALYWFT6L2UUX3AZP34FCTTKKXILANCNFSM43QCNQSQ>
> .
>
-- 
Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud
